src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -2334,7 +2334,9 @@ export function getChartSettingsMap(
     chartTypeSet: { [uniqueId: string]: ChartType },
     genomicChartSet: { [id: string]: GenomicChart },
     genericAssayChartSet: { [id: string]: GenericAssayChart },
-    clinicalDataBinFilter: { [uniqueId: string]: ClinicalDataBinFilter },
+    clinicalDataBinFilterSet: {
```

Comment:
I renamed this since it was initially named after the type that is gotten when you subscript it (which I found not totally accurate) to refer to; now that I think about it, map does seem more appropriate than set (maps have keys, sets don't)

Reply:
Additionally one such call to getChartSettingsMap() uses this._clinicalDataBinFilterSet as part of an argument

Reply:
Just reverted the map->set renames to their set names